cvm-attestation/src/AttestationProvider.py

```diff
@@ -194,6 +194,7 @@ def print_snp_platform_claims(self, encoded_token):
         self.log.info(f"Bootloader SVN: {claims['x-ms-sevsnpvm-bootloader-svn']}")
         self.log.info(f"Guest SVN: {claims['x-ms-sevsnpvm-guestsvn']}")
         self.log.info(f"Microcode SVN: {claims['x-ms-sevsnpvm-microcode-svn']}")
+        self.log.info(f"Firmware SVN: {claims['x-ms-sevsnpvm-snpfw-svn']}")

```

Comment:
"SNP Firmware SVN"?
Could you output the TEE SVN as well?

Reply:
Make a change as your comment. Thanks!